elisp-bytecode.texi

```diff
@@ -334,7 +334,15 @@ Discard one value.
 @node byte-dup
 @unnumberedsubsec @code{byte-dup} (137)
 @kindex byte-dup
-Duplicate one value.
+Push a copy the top of the evaluation stack
```

Comment:
I'd prefer "of the top...", or even "of the value at the top of"

Is it sufficiently clear this is not a copy in the sense of `copy-sequence`? (I think it is, actually).

Reply:
I have been shying away from using TOS which is the usual acronym used, but maybe we should start using that. How about the wording in 5c1160c ?